app/src/main/res/menu/activity_main.xml

```diff
@@ -5,5 +5,23 @@
         android:id="@+id/action_search"
         android:icon="@drawable/search"
         android:title="@string/title_find_place"
-        app:showAsAction="always" />
+        app:showAsAction="always"/>

```

Comment:
WARNING: [AlwaysShowAction] Prefer "`ifRoom`" instead of "`always`"
 Using `showAsAction="always"` in menu XML, or `MenuItem.SHOW_AS_ACTION_ALWAYS` in Java code is usually a deviation from the user interface style guide.Use `ifRoom` or the corresponding `MenuItem.SHOW_AS_ACTION_IF_ROOM` instead.

If `always` is used sparingly there are usually no problems and behavior is roughly equivalent to `ifRoom` but with preference over other `ifRoom` items. Using it more than twice in the same menu is a bad idea.

This check looks for menu XML files that contain more than two `always` actions, or some `always` actions and no `ifRoom` actions. In Java code, it looks for projects that contain references to `MenuItem.SHOW_AS_ACTION_ALWAYS` and no references to `MenuItem.SHOW_AS_ACTION_IF_ROOM`.